src/Sql/Sql.Test/Sql.Test.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.28.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.29.0-preview" />
```

Comment:
Note: New NuGet package has been published from the Azure SDK for .NET repository